scripts/A4.R

```diff
@@ -0,0 +1,35 @@
+# create ds object --------------------------------------------------------
+
+ds <- create_dataset(id = "A4")
+
+
+# download the data -------------------------------------------------------
+
+ds <- download_data(ds)
+
+# read in the spatial data ------------------------------------------------
+
+spatial_unit_df <- readr::read_csv("data/const/spatial_unit_postgres.csv")
+
+# data cleaning -----------------------------------------------------------
+
+### add spatial unit column "Switzerland"
+ds$data %>%
+  janitor::clean_names() %>%
+  tibble::as_tibble() %>%
+  dplyr::rename("beobachtungseinheit" = rubrik,
```

Comment:
"beobachtungseinheit" sounds too much like a unit: Maybe call it "verbrauchsart"

Reply:
implemented in new commit ✅